src/esp/bindings/GfxBindings.cpp

```diff
@@ -57,15 +69,16 @@ void initGfxBindings(py::module& m) {
   py::class_<Renderer, Renderer::ptr>(m, "Renderer")
       .def(py::init(&Renderer::create<>))
       .def("draw",
-           py::overload_cast<sensor::VisualSensor&, scene::SceneGraph&, bool>(
-               &Renderer::draw),
+           py::overload_cast<sensor::VisualSensor&, scene::SceneGraph&,
+                             RenderCamera::Flags>(&Renderer::draw),
            R"(Draw given scene using the visual sensor)", "visualSensor"_a,
-           "scene"_a, "frustumCulling"_a = true)
+           "scene"_a,
+           "flags"_a = RenderCamera::Flags{RenderCamera::Flag::FrustumCulling})
       .def("draw",
-           py::overload_cast<RenderCamera&, scene::SceneGraph&, bool>(
-               &Renderer::draw),
+           py::overload_cast<RenderCamera&, scene::SceneGraph&,
+                             RenderCamera::Flags>(&Renderer::draw),
            R"(Draw given scene using the camera)", "camera"_a, "scene"_a,
-           "frustumCulling"_a = true)
+           "flags"_a = RenderCamera::Flags{RenderCamera::Flag::FrustumCulling})
```

Comment:
This looks like what needs to be done:
```cpp
     .def(
          "draw",
          [](Renderer& self, sensor::VisualSensor& visualSensor,
             scene::SceneGraph& sceneGraph, RenderCamera::Flag flags) {
            self.draw(visualSensor, sceneGraph, RenderCamera::Flags{flags});
          },
          R"(Draw given scene using the visual sensor)", "visualSensor"_a,
          "scene"_a,
          "flags"_a = RenderCamera::Flag{RenderCamera::Flag::FrustumCulling})
      .def(
          "draw",
          [](Renderer& self, RenderCamera& camera,
             scene::SceneGraph& sceneGraph, RenderCamera::Flag flags) {
            self.draw(camera, sceneGraph, RenderCamera::Flags{flags});
          },
          R"(Draw given scene using the camera)", "camera"_a, "scene"_a,
          "flags"_a = RenderCamera::Flag{RenderCamera::Flag::FrustumCulling})
```

It's just the Flag enum that gets bound on the python side, not the whole flag set, so you have to write a wrapper lambda (Idk why @mosra didn't need to do a similar thing for the constructor on say the Phong shader but 🤷)

Reply:
Thanks, @erikwijmans this works. Appreciate the suggestion!

Reply:
I *did* need that -- the `py::init<>()` has `Flag`, not `Flags`. Since `Flag` is convertible to `Flags` this worked: https://github.com/mosra/magnum-bindings/blob/24b18aff931131f62ae635f1f5db6f1668d7a731/src/python/magnum/shaders.cpp#L59-L60

However for everything else except a constructor you need to write a wrapper lambda, as you did here.